ansible/library/conn_graph_facts.py

```diff
@@ -325,8 +332,16 @@ def get_host_connections(self, hostname):
         """
         return self.links.get(hostname)
 
-    def contains_hosts(self, hostnames):
-        return set(hostnames) <= set(self.devices)
+    def contains_hosts(self, hostnames, part):
+        if not part:
+            return set(hostnames) <= set(self.devices)
+        # It's possible that not all devices are found in connect_graph when using in devutil
+        THRESHOLD = 0.8
```

Comment:
Wondering what 80% found help? Maybe we could explicitly callout all found / partial / none found?

Reply:
The reason for I introduced partial found is not all devices defined in ```inventory``` are listed in ```conn_graph```. The default logic in ```find_graph``` in ```conn_graph_facts.py``` is attempting to find a ```xml``` file which contains all devices. So the logic will not work in ```devutils```. So I select the figure ```80%```, which means ```most``` devices are found.